homeassistant/components/fritz/__init__.py

```diff
@@ -54,11 +54,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await avm_wrapper.async_config_entry_first_refresh()
 
-    hass.data.setdefault(DOMAIN, {})
-    hass.data[DOMAIN][entry.entry_id] = avm_wrapper
+    hass.data.setdefault(DATA_KEY, {})
+    hass.data[DATA_KEY][entry.entry_id] = avm_wrapper
```

Comment:
Why not use runtime_data if the data is linked to the config entry?

Reply:
Agree, we should be using runtime data instead.

Reply:
for this, I first need to move the service setup into integrations async_setup ... will work on it :+1: